backend/src/main/java/com/bakdata/conquery/io/storage/xodus/stores/SerializingStore.java

```diff
@@ -114,21 +114,13 @@ public SerializingStore(XodusStoreFactory config, XodusStore store, Validator va
 
 		this.objectMapper = objectMapper;
 
-		valueWriter = objectMapper
-							  .writerFor(valueType)
-							  .withView(InternalOnly.class);
+		valueWriter = objectMapper.writerFor(valueType);
```

Comment:
Ist der jetzt global auf InternalOnly?

Reply:
nein, der wird in `com.bakdata.conquery.models.config.XodusStoreFactory#configureMapper` konfiguriert, das war davor leaky